tests/integration/it_location_admin_privileges_test.py

```diff
@@ -0,0 +1,136 @@
+from datetime import datetime
+import flask
+
+from timeless.access_control.manager_privileges import has_privilege
+from timeless.access_control.methods import Method
+from timeless.companies.models import Company
+from timeless.employees.models import Employee
+from timeless.restaurants.models import Location, Floor, Table
+
+
+def test_can_access_location_tables(app, db_session):
+    """User with Location Admin role can access the tables
+    from a Location owned by the company they work at"""
+    company = Company(
+        id=1, name="Foo Inc.", code="code1", address="addr"
+    )
+    location = Location(
+        id=1,
+        name="name",
+        code="123",
+        company_id=company.id,
+        country="US",
+        region="region",
+        city="city",
+        address="address",
+        longitude="123",
+        latitude="123",
+        type="type",
+        status="status"
+    )
+    floor = Floor (
+        id=1, description="1st Floor", location_id=location.id
+    )
+    table = Table(
+        id=1,
+        name="some table",
+        floor_id=floor.id,
+        x=40,
+        y=50,
+        width=320,
+        height=150,
+        status="available",
```

Comment:
Status is defined as an `int` in `timeless/restaurants/models.py`, this is causing the Travis CI and tests to break.